app/grandchallenge/algorithms/models.py

```diff
@@ -484,6 +502,16 @@ def public_test_case(self):
         except AttributeError:
             return False
 
+    def form_field_label(self):
+        title = f"{self.title}"
+        if self.active_image:
+            title += f" (Active image: {self.active_image.pk})"
+        if self.active_model:
+            title += f" (Active model: {self.active_model.pk})"
```

Comment:
Should we also include the user provided comment about the algorithm image and/or model?

Reply:
Good idea. Before the image and model pks would probably be best.